doc/getstarted.md

```diff
@@ -269,5 +254,199 @@ Top method improvements by size (bytes):
 3762 total methods with size differences.
 ```
 
-If `--csv <file_name>` or `--json <file_name>` is passed, all the diff data extracted and analyzed 
+If `--tsv <file_name>` or `--json <file_name>` is passed, all the diff data extracted and analyzed 
 will be written out for futher analysis.
+
+
+## Leveraging cloud resources
+
+CoreCLR uses a Jenkins CI instance to implement the quality bar checkins and inflight development. 
+This system defines a number of build jobs that produce artifacts that are cached for a peroid. 
+The cijobs tool allows for developers to easily access these artifacts to avoid rebuilding them 
+locally.
+
+The cijobs functionality is split into two different commands, list and copy. The first, `list` 
+allows for command line querying of job output.  The second, `copy`, copies built artifacts from 
+particular jobs from the cloud to the local machine.
+
+Here's the base help output:
+
+```
+$ cijobs --help
+usage: cijobs <command> [<args>]
+
+    list    List jobs on dotnet-ci.cloudapp.net for dotnet_coreclr.
+    copy    Copies job artifacts from dotnet-ci.cloudapp.net. Currently
+            hardcoded to dotnet_coreclr, this command copies a zip of
+            the artifacts under the Product sub-directory that is the
+            result of a build.
+``` 
+
+A common question for a developer might be "what is the last successful OSX checked build?"
+
+```
+$ cijobs list --match "osx"
+
+job checked_osx
+job checked_osx_flow
+job checked_osx_flow_prtest
+job checked_osx_prtest
+job checked_osx_tst
+job checked_osx_tst_prtest
+job debug_osx
+job debug_osx_flow
+job debug_osx_flow_prtest
+job debug_osx_prtest
+job debug_osx_tst
+job debug_osx_tst_prtest
+job release_osx
+job release_osx_flow
+...
+``` 
+
+The previous example shows searching for job names that match "osx".  The checked_osx jobs is the 
```

Comment:
Job names currently are semi-regular.  There's a "old style" and "new style" with the old sticking around because there are a number of flow jobs that would need to be changed over to the new style and that work isn't done.  But flow and prtest are uniform and I can add logic to drop them by default.